actin/operational/lookup_hash_for_actin-id

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
```

Comment:
Why put this in a separate file? Maybe just a function get you the same readability without needing to jump between files

Reply:
also the mix of underscores and hyphen in the script name looks weird

Reply:
> also the mix of underscores and hyphen in the script name looks weird

I didn't like either `actin_id` which fails to point out that an "ACTIN id" is a single thing, nor `actinid` which I find unreadable. I use this convention quite often and don't find it weird but I can see why you would say that.

In the interests of moving on I will use an underscore.

Reply:
> Why put this in a separate file? Maybe just a function get you the same readability without needing to jump between files

Simon asked specifically for this capability.

Reply:
> I didn't like either `actin_id` which fails to point out that an "ACTIN id" is a single thing, nor `actinid` which I find unreadable. I use this convention quite often and don't find it weird but I can see why you would say that.
> 
> In the interests of moving on I will use an underscore.

No worries if its a convention we are happy with